### ISSUE TITLE
Drag lock

### ISSUE THREAD

**Issue by morvael** [seastan/dragncards#105]
When I drag a card in free region it becomes undragabble. It's impossible to press mouse button. hold it and start dragging the card. Another card can be dragged normally. Once that card is dragged it becomes undraggable and the first card is draggable again.

Moving a card via code does not block its ability to be dragged, and even unblocks a blocked card (if it was the last moved).

I think it's related to menus showing behind the locked card.
![image](https://github.com/user-attachments/assets/60523327-d4dc-4780-9f0e-c0d9e43b742a)
